### PR TITLE
Support Koa v2

### LINKED ISSUE #27: Koa 2 Support
A new branch should support Koa 2 with `async` functions. Using `koa-convert` works in the meantime.

### PR DESCRIPTION
**Pull request by jgeurts**
Fixes #27

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "url": "https://github.com/koajs/koa-redis/issues"
   },
   "homepage": "https://github.com/koajs/koa-redis",
-  "version": "2.1.3",
+  "version": "3.0.0",
```
Comment (dead-horse) [inline package.json]: don't change version and history in PR, others LGTM

**Comment by dead-horse** [comment]
let travis only test in node 7 to make ci pass

File: benchmark/koa.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ if (process.argv[2] !== 'nosession') {
   }));
 }
 
-app.use(function *() {
+app.use(async () => {
```
Comment (DaAwesomeP) [inline benchmark/koa.js]: I'm not sure that you can use async with arrow functions

**Comment by DaAwesomeP** [comment]
If you take a look at Travis, it didn't actually pass without error:
https://travis-ci.org/koajs/koa-redis/jobs/214506109#L610

File: benchmark/koa.js (hunk 2 above)
Comment (pacofr) [inline benchmark/koa.js]: you can

Comment (DaAwesomeP) [inline benchmark/koa.js]: @pacofr look at Travis: https://travis-ci.org/koajs/koa-redis/jobs/214506109#L610

Comment (DaAwesomeP) [inline benchmark/koa.js]: Actually this might be an issue with istanbul

**Comment by DaAwesomeP** [comment]
@jgeurts any updates?

**Comment by dead-horse** [comment]
koa-redis already support koa@2.

**Pull request closed without merging**